### PR TITLE
Make it even clearer for beginners to replace home.ctp

### PR DESCRIPTION
**Pull request by bravo-kernel**
Incremental improvement to help out new users even better by informing them through newly added `ctp-warning` splash screen row. Refs https://github.com/cakephp/app/pull/319#issuecomment-169300334

really-help

### CODE REVIEW

**Comment by dereuromark** [comment]
I would have thought that the merged exception message suffices.

**Comment by bravo-kernel** [comment]
The exception requires users to look in the logs for the cause right? This informs them beforehand which feels even more helpful to me. Off to :sleeping: now.

**Comment by dereuromark** [comment]
I am still not sure who is actually deploying this boilerplate demo homepage on a server.
I mean, how useless is that? A website's most important (and only) start page is `/`.
Thus this would always be the first one to completely be written from scratch ;)
And even if you do, you will know who why you screwed up badly. But meeeh.

**Comment by jhli** [comment]
I like it.

**Comment by josegonzalez** [comment]
@dereuromark in my cake2 book, the first thing I do is show people how to deploy their apps, even before they make code changes, because I think continuous deployment and testing are so important. If you do that with a CakePHP app, the frontpage is a 404. Which then I have to explain why thats the case and instruct people to change it. So yes, I'm totally in favor of this change.

File: webroot/css/cake.css [hunk 1]
```diff
@@ -319,6 +319,11 @@ pre {
     margin-top:50px;
 }
 
+.checks.ctp-warning {
+    background-color: #ff5555; #bb0000;	#f2dede;
```
Comment (markstory) [inline webroot/css/cake.css]: What's the deal with 3 background colors?

**Comment by bravo-kernel** [comment]
Totally looked over the color codes, removed and rebased mr. @markstory,

**Comment by ionas** [comment]
Thank you!

**Comment by ADmad** [comment]
I'll just leave this here: http://stackoverflow.com/questions/36116483/please-be-aware-that-this-page-will-not-be-shown-if-you-turn-off-debug-mode-unle

**Comment by lorenzo** [comment]
:(

**Comment by lorenzo** [comment]
Can remove it?

**Comment by markstory** [comment]
Lets leave it in. At least with the current state people who read the page won't go to stackoverflow.

**Comment by lorenzo** [comment]
Maybe change the color to something that does not look as an error?

**Comment by bravo-kernel** [comment]
Never thought I would reach the point where I would not be able to empathize with the new user anymore :scream:

**Comment by jhli** [comment]
• Yellow/orange instead of red as it is a notice/warning?
• "... Until you replace View/Pages/home.ctp"?

Anyway this won't help the user @Admad's SO post. For those all we can do is explaining dipatching/routing/mvc and posdibly .htaccess and/or how/where/if to place third party public standalone webapps like phpmyadmin; all stuff for the docs.

**Comment by markstory** [comment]
@lorenzo Yellow might work better.

**Pull request merged**